src/telev/telev.c

```diff
@@ -341,6 +341,12 @@ int telev_poll(struct telev *tel, bool *marker, struct mbuf *mb)
 }
 
 
+bool telev_is_empty(struct telev *tel)
```

Comment:
input parameter can be `const`

please check for a valid `tel` pointer. If tel is NULL then return `false`

Reply:
maybe better should return `true` if tel is NULL.